common/src/PreferenceManager.cpp

```diff
@@ -588,6 +588,12 @@ namespace TrenchBroom {
     }
 
     std::map<IO::Path, QJsonValue> migrateV1ToV2(const std::map<IO::Path, QJsonValue>& v1Prefs) {
```

Comment:
This `migrateV1ToV2` is only run when migrating the wxWidgets preference (TB 2019.6 and earlier - was stored in the registry on Windows) to the new JSON format used in 2020.1 and later. We don't have a system for migrating the preference key.